src/SampleConnector.csproj

```diff
@@ -224,6 +238,9 @@
       <HintPath>packages\protobuf-net.Core.3.0.101\lib\net461\protobuf-net.Core.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="PubnubPCL, Version=6.19.5.0, Culture=neutral, PublicKeyToken=dc66f52ce6619f44, processorArchitecture=MSIL">
```

Comment:
Hi @lotaked , same here. PubnubPCL 6.17.0 is used in FDX SDK, Are our packages requesting 6.19.5?

Reply:
Thank you for pointing that out! It was automatically updated by Visual Studio when updating the SDK. I've corrected it now.